src/main/resources/assets/minecolonies/gui/windowhutguardtower.xml

```diff
@@ -66,17 +66,14 @@
                    label="$(com.minecolonies.coremod.gui.workerhuts.mobsList)"/>
             <button id="recalculate" size="100 14" pos="45 29"
                     label="$(com.minecolonies.coremod.gui.workerhuts.calculate)"/>
-            <list id="mobs" size="164 135" pos="13 48">
+            <list id="mobs" size="164 165" pos="13 48">
                 <box size="100% 30" linewidth="2">
                     <button id="prioUp" size="15 15" label="↥" pos="0 0"/>
                     <button id="prioDown" size="15 15" label="↧" pos="0 15"/>
                     <label id="name" size="120 12" pos="35 16" textalign="MIDDLE_LEFT" color="black"/>
                     <button id="switch" size="100 13" pos="5 1" align="TOP_RIGHT" label="$(com.minecolonies.coremod.gui.workerhuts.retrieveOn)"/>
                 </box>
             </list>
-            <buttonimage id="getTool" size="129 17" pos="30 200" textcolor="black"
-                         source="minecolonies:textures/gui/builderhut/builder_button_medium_large.png"
-                         label="$(com.minecolonies.coremod.gui.guardtower.getTool)"/>
```

Comment:
Are you sure that setTarget and getTool buttons are linked to the same button handler in the code?

Reply:
All getTool does is give the player the tool. setTarget seems to do more

Reply:
Probably good to keep it for now. Also the get tool way on page 2, is not always visible only if manual patrolling or guarding is selected

Reply:
Those two options are the only way the tool is *useful* iirc

Reply:
But sure I'll add it back